mms/service_manager.py

```diff
@@ -119,6 +119,9 @@ def load_model(self, model_name, model_dir, manifest, model_service_class_def, g
         batch_size : int
             batch size
         """
+        # Check which manifest to pass down
+        manifest = manifest if manifest_legacy is None else manifest_legacy
```

Comment:
If manifest legacy is used the same way as manifest, you may want to move this line to ModelLoader and only pass the existing manifest file through all the various calls.  It would be much simpler and require less code.  But, does legacy not require special handling?